course_content/iris_course/3.Subcube_Extraction.ipynb

```diff
@@ -534,7 +534,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "1\\. Load the `sea_water_potential_temperature` cube from the file in ``iris.sample_data_path('atlantic_profiles.nc')`` and print the cube list. Store these cubes in a variable called `cube`."
```

Comment:
this must have just been a copy and past error as we are loading in a cube not a cubelist.